Module3_Assignment1/Module3_Assignment1_Part1.cls

```diff
@@ -0,0 +1,46 @@
+public class Module3_Assignment1_Part1 {
+    public static List<SObject> studentInClass(ID classId){
```

Comment:
As it is returning list of students which can be multiple so name should be studentsInClass(). Also return type can be List<Student__c> as well